service-jehlomat/src/main/kotlin/services/Mailer.kt

```diff
@@ -164,4 +169,19 @@ class Mailer: MailerService {
         println(response.status)
         println(response.data)
     }
+
+    @Throws(MailjetException::class)
+    override fun sendPassResetEmail(email: String, userId: Int, resetUrlCode: String) {
+         val request = MailjetRequest(Emailv31.resource)
+             .property(
+                 Emailv31.MESSAGES, prepareBodyWithLink(
+                     3884047,
+                     "Požadavek na reset hesla",
+                     "${publicUrl}uzivatel/heslo/?userId=${userId}&code=${resetUrlCode}",
```

Comment:
Prosím tě, máme endpoint pro získáný detailu uživatele bez autorizace? Nový endpoint `/test-code` vrací jen 204 NoContent, ale k tomu, aby si uživatel mohl nastavit heslo, tak v payload musí být i email.
Pokud ne, budu to muset udělat tak, že uživatel musí znovu zadat email i nové heslo, což není moc hezké uživatelského hlediska.

Reply:
Nemáme a ani nemůžeme mít, to by byl bezpečnostní problém. 

Email ale není pro reset hesla potřeba, podívej se na swagger nebo na jednotlivé request objekty. Na základě tvých připomínek ohledně GDPR jsem to předělal na kód a userId.